src/fidgets/farcaster/components/CastRow.tsx

```
@@ -18,6 +18,15 @@ import {
   HeartIcon,
 } from "@heroicons/react/24/outline";
 import { HeartIcon as HeartFilledIcon } from "@heroicons/react/24/solid";
+import {
+  publishReaction,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'publishReaction'. Consider to delete it or rename it.

'publishReaction' is defined here:


(lint/suspicious/noRedeclare)